test/ssl_test.c

```diff
@@ -233,7 +233,9 @@ static int execute_test(SSL_TEST_FIXTURE fixture)
         client_ctx = SSL_CTX_new(DTLS_client_method());
         if (test_ctx->handshake_mode == SSL_TEST_HANDSHAKE_RESUME) {
             resume_server_ctx = SSL_CTX_new(DTLS_server_method());
-            OPENSSL_assert(resume_server_ctx != NULL);
+            resume_client_ctx = SSL_CTX_new(DTLS_client_method());
+            OPENSSL_assert(resume_server_ctx != NULL
```

Comment:
as a personal style issue, I prefer a single test inside an assert.  so adding the resume_client_ctx would be cut-and-paste of the pre-existing two lines, and having two asserts one for each condition.  But if don't want to do that, it's okay.